email_alert_service/models/message_processor.rb

```diff
@@ -14,7 +14,7 @@ def process(document_json, properties, delivery_info)
 
     acknowledge(message)
   rescue MalformedDocumentError => e
-    Airbrake.notify_or_ignore(e)
+    GovukError.notify(e.message)
```

Comment:
If you pass in `e` here Sentry will show metadata from the error as well.

Reply:
Done, nice feature!